### PR TITLE
Bring back okta auth

### PR DESCRIPTION
**Pull request by yeameen**
Moving to older version of jetty to resolve dependency conflicts.

**Verification**
- ` mvn clean package -DskipTests` successful
- login with okta successful for presto-cli

### CODE REVIEW

**Comment by yeameen** [comment]
👀 @JamesRTaylor @bearcage @bhuwanchopra

File: presto-cli/pom.xml [hunk 1]
```diff
@@ -93,6 +98,17 @@
...
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-server</artifactId>
+            <version>9.4.14.v20181114</version>
```
Comment (yeameen) [inline presto-cli/pom.xml:109]: Using an older version to resolve conflicts

**Comment by bearcage** [comment]
lgtm as long as @JamesRTaylor is good with it too

**Comment by JamesRTaylor** [comment]
👍 

Would be important to think about how to encapsulate the Lyft-specific part of this PR and contribute back the rest to OSS presto. Otherwise, it makes it more difficult to move to newer versions of presto.

**Comment by yeameen** [comment]
> 👍
> 
> Would be important to think about how to encapsulate the Lyft-specific part of this PR and contribute back the rest to OSS presto. Otherwise, it makes it more difficult to move to newer versions of presto.

Right now it has several Lyft specific configs & code. I am going to merge it (to unblock prestoproxy) and created a task to make this more generic - https://jira.lyft.net/browse/DATA-9478

**Pull request merged**